_docs/6_BlockArchives/README.md

```diff
@@ -0,0 +1,20 @@
+# Block archives
+
+## Create chain archive
+TODO: add section
+
+## Update archives
+In order to keep existing archives up-to-date you can create a simple setup with Docker Compose:
+```bash
+mkdir update-archives
+cd update-archives
+touch docker-compose.yml
+```
+
+Sample [docker-compose.yml](docker-compose.yml) can be used as a reference.
+
+Pull fresh image and start containers:
+```bash
+docker-compose pull
+ONFINALITY_API_KEY="" docker-compose up -d
```

Comment:
I believe it supports some kind of env file, so you don't forget about environment variable accidentally